src/mentionui.js

```diff
@@ -61,6 +70,55 @@ export default class MentionUI extends Plugin {
 		this._mentionsConfigurations = new Map();
 
 		editor.config.define( 'mention', { feeds: [] } );
+
+		this._requestFeedDebounced = debounce( ( marker, feedText ) => {
+			this._getFeed( marker, feedText )
+				.catch( error => {
```

Comment:
Why do you define `catch()` before `then()`? For once, it's the opposite to how you think about error handling (1. do things, 2. handle errors). Plus, see "Exhibit A".

Reply:
I've refactored the MentionUI to events... dunno why I didn't that before - it is much better now IMO.